metricflow/test/model/validations/test_measures.py

```diff
@@ -332,3 +336,88 @@ def test_count_measure_with_distinct_expr(simple_model__pre_transforms: UserConf
     assert (
         actual_error.find(expected_error_substring) != -1
     ), f"Expected error {expected_error_substring} not found in error string! Instead got {actual_error}"
+
+
+def test_percentile_measure_missing_agg_params(simple_model__pre_transforms: UserConfiguredModel) -> None:
```

Comment:
We should also add some tests for `MEDIAN` agg

Reply:
I can't really think of any test cases that go wrong based on this validation case specifically. The median agg doesn't have any specific validations attached to it, and, just like the sum agg bad_measure_3, would raise an error if any agg_params are provided, which i think is what we want?

Reply:
Yep, that is exactly what I thought: it should raise an error if `agg_params` is provided.

Reply:
The post-validation transform rules are all applied pre-validation. Yes, I know. But that's how it works.

We should assert that the model input doesn't have wacky values, but that has to happen in the transform rule.